README.md

```diff
@@ -288,7 +288,7 @@ If you try to read from a stream and pipe to a new file, you will obtain an empt
 
 ## JSON Schema body validation
 
-If you enable `attachFieldsToBody: 'keyValues'` then the response body and JSON Schema validation will behave similarly to `application/json` and [`application/x-www-form-urlencoded`](https://github.com/fastify/fastify-formbody) content types. Files will be decoded using `Buffer.toString()` and attached as a body value.
+If you enable `attachFieldsToBody: 'keyValues'` then the response body and JSON Schema validation will behave similarly to `application/json` and [`application/x-www-form-urlencoded`](https://github.com/fastify/fastify-formbody) content types. Files will be decoded using `Buffer` and attached as a body value.
```

Comment:
I know this whole README is full of "you"s and all, but one commit at a time, we need to do better 🤣

```suggestion
When the `attachFieldsToBody` parameter is set to `'keyValues'`, the response body and JSON Schema validation will behave in a manner similar to that observed with the `application/json` and [`application/x-www-form-urlencoded`](https://github.com/fastify/fastify-formbody) content types. Additionally, files will be attached to the body as Buffer objects.
```

Reply:
Ref: https://fastify.dev/docs/latest/Guides/Style-Guide

Reply:
Good call out. Just updated

Reply:
Thanks! 🚀